Assets/Scripts/Tools/NodeTool.cs

```diff
@@ -74,6 +93,28 @@ public void OnRightClick(Transform cameraTransform, bool isRayCastHit, RaycastHi
       contextMenuHandler.OpenContextMenu(node);
     }
 
-    public void OnSelect() => _toolPanelController.SetBackgroundColor(Color.green);
+    public void OnSelect()
+    {
+      _shouldGhostNodeBeActive = true;
+      if (_toolPanelController != null)
```

Comment:
why are those null checks here?

Reply:
because ToolController invokes OnSelect() before Start() method in NodeTool is executed